### PR TITLE
ControlInterface go-http-api optout

### PR DESCRIPTION
**Pull request by gsvr**
Currently the opt-out link for subscriber edits just makes subscriptions inactive, it should also opt out the user.

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: controlinterface/tests.py [hunk 1]
```diff
@@ -254,7 +273,7 @@ def test_subscription_cancel_optout_all(self):
             to_addr="+271112", active=True).count()
         self.assertEqual(activebefore, 1)
         response = self.client.post(
-            reverse('controlinterface.views.subscription_edit'), cancelform)
+            reverse('controlinterface.views.subscription_edit'), optoutform)
```
Comment (imsickofmaps) [inline controlinterface/tests.py:274]: needs an assert for checking the responses were used

File: controlinterface/tests.py [hunk 2]
```diff
@@ -114,6 +117,9 @@ def setUp(self):
             self.username,
             'testuser@example.com', self.password)
         self.client = Client()
+        self.optout_client = OptOutsApiClient(
```
Comment (imsickofmaps) [inline controlinterface/tests.py]: remove this, not used

**Comment by imsickofmaps** [comment]
Minor tweaks :+1: when addressed and Travis happy

**Pull request merged**